docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -198,6 +198,42 @@ IMPORTANT: You can only configure these settings on nodes with the
 <<data-frozen-node,`data_frozen`>> role. Additionally, nodes with a shared
 cache can only have a single <<path-settings,data path>>.
 
+[discrete]
+[[searchable-snapshots-costs]]
+=== Reducing running costs with {search-snaps}
+
+Using {search-snaps} can significantly reduce the running costs of your {es}
+cluster. Searchable snapshot indices do not need to be replicated for
+resilience since {es} will recover any missing shards from the snapshot
+repository after a node failure. In contrast, regular indices must always be
+replicated to multiple nodes to ensure resilience. This means that using
+{search-snaps} reduces the number of shard copies that you need for your
+infrequently-accessed data by a factor of two. Your cold data tier therefore
+needs half of the disk space and half of the number of nodes that they would
+need if not using {search-snaps}. The partially-mounted indices in your frozen
+tier will need even fewer resources.
+
+Furthermore when a fully-mounted {search-snap} index is mounted or relocated
+its contents are copied from the repository rather than from another node in
+your cluster. Retrieving data from the snapshot repository is usually very
+cheap. In contrast, the contents of regular indices are copied from another
+node in the cluster. Transferring data from a node in a different zone often
+carries a significant cost.
+
+NOTE: You can realise these cost savings in most environments, including on all
+major cloud platforms, but take note that they do not apply to every
+environment. For example, if retrieving data from your snapshot repository
+carries a high cost then you may find {search-snaps} to be more expensive than
+regular indices. Ensure that the cost structure of your operating environment
+is compatible with {search-snaps} before using them.
+
+WARNING: Most cloud providers charge significant fees for data transferred
+between regions and for data transferred out of their platforms. You should
+only mount snapshots into a cluster that is in the same region as the snapshot
+repository. If you wish to search data across multiple regions, configure
+multiple clusters and use <<modules-cross-cluster-search,{ccs}>> or
+<<xpack-ccr,{ccr}>> instead of {search-snaps}.
```

Comment:
I tried to restructure this so that:

* Users can get a high-level takeway from the first paragraph
* The information about replicas and repo data retrieval is in separate sections

I think this makes it a little easier to parse, but the copy you current have is fine.

```suggestion
[discrete]
[[searchable-snapshots-costs]]
=== Reduce costs with {search-snaps}

Before using {search-snaps}, ensure they're cost effective in your environment.
In most cases, {search-snaps} reduce the costs of running a cluster by removing
the need for replica shards. However, if it's particularly expensive to retrieve
data from a snapshot repository, {search-snaps} may be more costly than
regular indices.

[discrete]
[[replica-costs]]
==== Replica costs

For resiliency, a regular index requires multiple replica shards across multiple
nodes. If a node fails, {es} uses these replicas to recover any missing data.
A {search-snap} index doesn't require replicas. If a node containing a
{search-snap} index fails, {es} can recover missing data from the snapshot
repository.

Without replicas, {search-snap} indices require far fewer resources. A cold data
tier that contains only fully-mounted {search-snap} indices requires half the
nodes and disk space of a tier containing regular indices. The frozen tier,
which contains only partially-mounted {search-snap} indices, requires even fewer
resources.

[discrete]
[[snapshot-retrieval-costs]]
==== Snapshot retrieval costs

// tag::search-snap-costs[]
When {es} mounts or reallocates a fully-mounted {search-snap} index, it copies
the index's data from the snapshot repository. The cost of retrieving this data
is typically low but may be higher in some environments.
// end::search-snap-costs[]

Many cloud providers charge significant fees for data transfers between regions
or transfers out of their platforms. To avoid these fees, only mount
{search-snap} indices on clusters in the same region as the snapshot repository.
To search across multiple regions, use <<modules-cross-cluster-search,{ccs}>> or
<<xpack-ccr,{ccr}>> instead.
```